package.json

```diff
@@ -11,7 +11,7 @@
     "compile:css": "yarn clean-dist && cleancss -o dist/compressed.css node_modules/material-components-web/dist/material-components-web.css node_modules/dialog-polyfill/dialog-polyfill.css src/libs/css/app.css",
     "dev": "yarn compile:dev && yarn start:dev",
     "lint": "eslint src/libs --ext=js --ext=jsx && eslint src/demos --ext=js --ext=jsx",
-    "test": "jest",
+    "test": "yarn run lint && yarn run jest",
```

Comment:
no, test should be test. Fix the travis ci config if you want to run both commands.

Reply:
Ok I agree, but should we add it to lint command ?

Reply:
because you want a command to run both at the same time? maybe "check" then